src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1948,7 +1949,13 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                         }
                         isVisibleResponse = false;
                     }
-
+                }
+                
+                if (question.questionType == FeedbackQuestionType.CONTRIB) {
```

Comment:
I see, I didn't notice that this was nested in the branch for a single question results. The logic is correct then, but I still think it might not be good for logic to need to specifically check for the question's type and have behaviour specific to Contribution questions. Will it be possible to use the roster's data in <code>bundle.rosterTeamNameMembersTable</code> to get the team names in FeedbackContributionQuestionDetails instead?